views/partials/header.ejs

```diff
@@ -1,4 +1,26 @@
-<header>
+<h1>The Jobs EJS Application</h1>
+<% if (user) { %>
+   <p>User <%= user.name %> is logged on.</p>
+   <form method="POST" action="/sessions/logoff">
+   <button>Logoff</button>
+   </form>
+<% } %>
+<% if (errors) {
```

Comment:
Requires change:
Starting at this line and the below are missing applied changes from the week13 assignment.